### ISSUE TITLE
Shall we add support of uintptr and Pointer?

### ISSUE THREAD

**Issue by tisonkun** [uber-go/atomic#88]
I'll be glad to work on it if it is good to go.

```go
func LoadUintptr(addr *uintptr) (val uintptr)
func StoreUintptr(addr *uintptr, val uintptr)
func AddUintptr(addr *uintptr, delta uintptr) (new uintptr)
func SwapUintptr(addr *uintptr, new uintptr) (old uintptr)
func CompareAndSwapUintptr(addr *uintptr, old, new uintptr) (swapped bool)

func LoadPointer(addr *unsafe.Pointer) (val unsafe.Pointer)
func StorePointer(addr *unsafe.Pointer, val unsafe.Pointer)
func SwapPointer(addr *unsafe.Pointer, new unsafe.Pointer) (old unsafe.Pointer)
func CompareAndSwapPointer(addr *unsafe.Pointer, old, new unsafe.Pointer) (swapped bool)
```

**Comment by abhinav** [comment]
Yes, this sounds reasonable. We would welcome a PR for this! Although for unsafe.Pointer, I would be reluctant to name it `atomic.Pointer`. Maybe we can name it `UnsafePointer` (used as `atomic.UnsafePointer`) to make it clear this is still an unsafe API.

**Comment by tisonkun** [comment]
@abhinav So here is the pull request #90

**Comment by tisonkun** [comment]
@abhinav is there a release cycle or trigger description. I may hurry a bit but it is the nature a developer want to know how and when the work released :P

**Comment by abhinav** [comment]
Hey @tisonkun, no there's no fixed release cycle, but we can tag a release this week. Thanks for your contribution!

**Comment by abhinav** [comment]
@tisonkun Change released in v1.8.0

**Comment by tisonkun** [comment]
@abhinav thanks!
